alicuda/alidock.cc7/README.md

```diff
@@ -0,0 +1,3 @@
+`alisw/alidock`-based container to support nvidia CUDA runtime and cross-compilation.
+
+See main project at [alidock](htpps://github.com/dberzano/alidock)
```

Comment:
Please update the link to alidock. Maybe add a title with the container name as you would pull it from Docker:

```markdown
alipier/alicuda:alidock.cc7
===========================
```